Tetfuza/TetfuzaBackend/TetfuzaBackend.cs

```diff
@@ -89,70 +69,27 @@ private int DropSpeed
                 return dropFrameDelay;
             }
         }
-
-        private List<char> EmptyLine
-        {
-            get
-            {
-                return BoardLine(EMPTY_CHAR);
-            }
-        }
         
         /// <summary>
         /// Creates a new instance of a TetfuzaBackend, and initializes an empty board
         /// </summary>
-        public TetfuzaBackend()
+        public TetfuzaBackend(IConsole keyboard, IDisplay screen)
         {
-            Board = BoardInit();
-            _keyboard = new InputChecker(new SystemConsole());
+            Board = new TetfuzaBoard();
```

Comment:
Should it be _board bc it’s private?

Reply:
You're right. Board used to be a property, but I converted it to a private member after I made the board class, and forgot to change the naming convention.